### PR TITLE
[User] Sign Up

### PR DESCRIPTION
**Pull request by anhlh-3049**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/48969

## WHAT (optional)
- Form Sign Up
- Sign Up

## Evidence (Screenshot or Video)
Screenshot_2

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: app/assets/stylesheets/user/style.scss [hunk 1]
```diff
@@ -1285,6 +1285,25 @@ input::-ms-input-placeholder {
         margin-bottom: 30px;
     }
 } 
+.form {
+    width: 60%;
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/user/style.scss]: tab 2 space

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  def show; end
+
+  def create
+    @user = User.create user_params
+    if @user.save
+      flash[:success] = "Registration success"
+      redirect_to root_path
+    else
+      flash[:danger] = "Registration error"
+      render "new"
```
Comment (ngocvt-0484) [inline app/controllers/users_controller.rb]:
```suggestion
      render :new 
```

File: app/views/users/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div class="form">
+<%= form_for @user do |f| %>
+  <%= f.label "Name" %>
+  <%= f.text_field :name, class: "form-control" %>
+
+  <%= f.label "Email" %>
```
Comment (ngocvt-0484) [inline app/views/users/_form.html.erb]: i18n

**Comment by anhlh-3049** [comment]
ready

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = "Royal Hotel"
```
Comment (ngocvt-0484) [inline app/helpers/application_helper.rb]: i18n

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    <p>
+        <%= t ".service_html" %>
+    </p>
+  </div>
+  <div class="col-md-12">
+    <div class="login-form">
+      <%= render "form" %>
+    </div>
+  </div>
+</div>
```
Comment (ngocvt-0484) [inline app/views/users/new.html.erb:18]: indent

**Comment by anhlh-3049** [comment]
ready

**Pull request merged**